addon/components/au-accordion.hbs

```diff
@@ -4,8 +4,9 @@
     @reverse={{@reverse}}
     {{on 'click' this.openAccordion}}
     data-test-accordion-toggle
+    as |Group|
```

Comment:
Not sure if `ToolbarGroup` is a better name. Users can still name it like that, but most people will use the documented name I think.